### PR TITLE
Feature/addbutton change

### PR DESCRIPTION
**Pull request by pusztaienike**
It is working what I expected, but we have 2 differences compare to the previous version:
- On **Content Types** root we have no action with the name of 'Add'
(https://dev.demo.sensenet.com/odata.svc/Root/System/Schema/ContentTypes/GenericContent/('ListItem')/Actions) so here Add button is not enabled  <==> In the previous version there was an Add button there
- On **Setup** root we have now Add button, cause we have 'Add' action here
(https://dev.demo.sensenet.com/odata.svc/Root/System/('Settings')/Actions) and we have so many child types what we can add: 
(https://dev.demo.sensenet.com/odata.svc/Root/System/('Settings')/EffectiveAllowedChildTypes?$select=Name&metadata=no) <==> In the previous version we did not have Add button on Setup

### CODE REVIEW

**Comment by herflis** [comment]
@pusztaienike sorry, I forgot to mention that in case of ContentTypes the Action that should be available is not 'Add' but 'Upload'

**Comment by pusztaienike** [comment]
> @pusztaienike sorry, I forgot to mention that in case of ContentTypes the Action that should be available is not 'Add' but 'Upload'

For Content type I had to track the queryData cause it has an allowedChildTypes attribute, what I needed for the Action list... 
Changes have been made. It is working now for Content types as well

File: apps/sensenet/src/hooks/use-drawer-items.tsx [hunk 1]
```diff
@@ -60,6 +61,32 @@ export const useDrawerItems = () => {
     [localization],
   )
 
+  const getRootFromSetting = useCallback(
+    (item: ContentDrawerItem | QueryDrawerItem | BuiltinDrawerItem | DashboardDrawerItem) => {
+      switch (item.itemType) {
```
Comment (B3zo0) [inline apps/sensenet/src/hooks/use-drawer-items.tsx]: This should be added to the PersonalSettings.ts defaultSettings object.

File: apps/sensenet/src/components/drawer/PermanentDrawer.tsx [hunk 2]
```diff
@@ -63,28 +74,25 @@ const PermanentDrawer: React.FunctionComponent<RouteComponentProps> = props => {
...
-            ) : (
+            return (
               <NavLink
                 to={`/${btoa(repo.configuration.repositoryUrl)}${item.url}`}
                 activeStyle={{ opacity: 1 }}
                 style={{ textDecoration: 'none', opacity: 0.54 }}
-                key={index}>
-                <ListItem button={true}>
+                key={index}
+                isActive={match => {
```
Comment (B3zo0) [inline apps/sensenet/src/components/drawer/PermanentDrawer.tsx]: I think this prop is not needed anymore.

File: apps/sensenet/src/components/drawer/PermanentDrawer.tsx [hunk 3]
```diff
@@ -121,11 +129,20 @@ const PermanentDrawer: React.FunctionComponent<RouteComponentProps> = props => {
                 />
                 <ListItemSecondaryAction>
                   {device === 'mobile' ? null : (
-                    <Link to={`/personalSettings`} style={{ textDecoration: 'none' }}>
+                    <NavLink
+                      to="/personalSettings"
+                      style={{ textDecoration: 'none' }}
+                      isActive={match => {
+                        if (!match) {
```
Comment (B3zo0) [inline apps/sensenet/src/components/drawer/PermanentDrawer.tsx]: Please put the setCurrentPath logic to the onChange prop and remove this prop, also do this for the other navlink below.

**Pull request merged**